packages/fxa-payments-server/src/lib/formats.ts

```diff
@@ -1,11 +1,108 @@
-import dayjs from 'dayjs';
+import { FluentNumber, FluentDateTime } from '@fluent/bundle';
 
-export function formatCurrencyInCents(amount: number): string {
-  const decimal = (amount / 100.0).toFixed(2);
-  return `$${decimal}`;
+/**
+ * The following function are for Currency localization
+ */
+
+const baseCurrencyOptions = {
+  style: 'currency',
+  currencyDisplay: 'symbol',
+};
+
+/**
+ * This method is used to provide Fluent with a localizable value that can be formatted per .ftl file based on localization requirements
+ *
+ * @param amountInCents
+ * @param currency
+ */
+export function getLocalizedCurrency(
+  amountInCents: number,
+  currency: string
+): FluentNumber {
+  const decimal = amountInCents / 100;
+  const options = { ...baseCurrencyOptions, currency };
+
+  return new FluentNumber(decimal, options);
```

Comment:
This looks great, 👍 . I'll work on protecting the `currency` option in `@fluent/bundle` tomorrow.

Reply:
I've just published `@fluent/bundle` 0.15.1 which [forbids some formatting options](https://github.com/projectfluent/fluent.js/blob/master/fluent-bundle/CHANGELOG.md#fluentbundle-0151-april-7-2020), like `currency`, in the `NUMBER` and `DATETIME` functions.

Reply:
I'll update, thanks :)